Makefile

```diff
@@ -97,10 +99,8 @@ package: promscale.control ## Generate extension artifacts for packaging
 	cargo pgx package --pg-config ${PG_CONFIG}
 
 .PHONY: install
-install: promscale.control ## Install the extension in the Postgres found via pg_config
-	cargo pgx schema pg${PG_BUILD_VERSION} --out sql/promscale--${EXT_VERSION}.sql --release
-	bash create-upgrade-symlinks.sh
-	cargo pgx install --pg-config ${PG_CONFIG}
+install: ## Install the extension in the Postgres found via pg_config
```

Comment:
don't we need to build it first?

Reply:
Yes, but `make install` only installs. You have to run `make package` first (as in the instructions).

Reply:
Is there a downside to making a dependency chain like: `install` -> `/target/release/promscale-pg${PG_BUILD_VERSION}` -> `package`? Non-sudo `make install` is sometimes useful for local development.